exercises/delia_solution/leap_numbers/leap.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func checkIsLeapYear(year int) bool {
+	leap := false
+	if year%4 == 0 {
+		if year%100 == 0 {
+			if year%400 == 0 {
+				leap = true
+			} else {
+				leap = false
+			}
+		} else {
+			leap = true
+		}
+	}
+	return leap
+}
+
```

Comment:
Maybe the conditions could've been grouped easier instead of having these nested ifs

Reply:
I agree that this can be improved but I don't think it's wrong